pandas/core/construction.py

```diff
@@ -278,3 +312,241 @@ def array(
 
     result = PandasArray._from_sequence(data, dtype=dtype, copy=copy)
     return result
+
+
```

Comment:
+1 on combing these routines *into* array module

Reply:
sure

Reply:
though im not wild about that name (hard to grep for)